solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/Cell/RepositoryViewCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RepositoryViewCell.swift
+//  GitHubApp
+//
+//  Created by Felipe on 06/07/22.
+//
+
+import UIKit
+
+final class RepositoryViewCell: UITableViewCell {
+    
+    struct RepositoryViewCellConfiguration{
```

Comment:
Somente um nit de espaçamento aqui:
```suggestion
    struct RepositoryViewCellConfiguration {
```